README.md

```diff
@@ -17,20 +17,26 @@ The game we've created is Deny and Conquer. Deny and Conquer is a strategic mult
 
 ## Description of Design
 
-**include high level description of the design**
+Deny & Conquer is designed using Java for Backend Design and using JavaFX for the Frontend Design. The game uses sockets for client-server communication. Clients connect to the server using TCP sockets, allowing bidirectional communication between the clients and the central game server, where the server starts listening for incoming connections from clients. When a client wants to join the game, it establishes a TCP connection to the server by connecting to the server's IP address and port number. Upon successful connection, clients register with the server by providing their player information, such as player name and pen color. The server assigns a unique ID to each player to distinguish them.
```

Comment:
Deny & Conquer is designed using Java for Backend Design and using JavaFX for the Frontend Design.

Don't need to use **Design** many times. 
Deny & Conquer is designed using Java for Backend and using JavaFX for the Frontend.